src/core/core-varlink.c

```diff
@@ -120,7 +129,13 @@ int manager_varlink_send_managed_oom_update(Unit *u) {
         if (r < 0)
                 return r;
 
-        return varlink_notify(u->manager->managed_oom_varlink_request, v);
+        if (u->manager->managed_oom_varlink_request)
+                r = varlink_notify(u->manager->managed_oom_varlink_request, v);
+
+        if (u->manager->managed_oom_varlink)
+                r = varlink_send(u->manager->managed_oom_varlink, "io.systemd.oom.SendManagedOOMCGroups", v);
```

Comment:
the method name is misleading. It suggests that oomd would send something, but it's of course the service manager that sends the change notification here.

maybe call the function ReportManagedOOMCGroupChange() or so?

Reply:
Renamed to ReportManagedOOMCGroups to be consistent with SubscribeManagedOOMCGroups in naming